CONTRIBUTING.rst

```diff
@@ -53,6 +53,8 @@ Workflow
 ********
 
 * This project follows the `gitflow`_ branching model. Please name your branch accordingly.
+   - Project maintainers first create a PR to merge a release branch into the ``develop`` branch, then they merge the release branch into the ``master`` branch locally and push to master.
+   - It helps to think of release branches as a staging area. Bugfixes meant for a specific release branch are merged through PRs.
```

Comment:
This one is a little less clear.  To me it seems logical that bug fixes would be made in the same branch as the bugs, so it's confusing that it would need to be said. Is the emphasis on that they should be made through PRs rather than direct commits?  I also don't get a whole lot out of the staging area comment.

Reply:
> Is the emphasis on that they should be made through PRs rather than direct commits?

Yeah, it's mainly that and partially to emphasize that if someone's submitting a bugfix for a feature that's on a release branch, that fix should be PR'ed to the release branch instead of develop where the bug also exists.

Reply:
It might be good to say that all the way out, like "release branch are merged through PRs to that branch, not to ``develop``." I would also reverse the order of these two statements. Put the directive of what to do first, and then the explanation of why.